### ISSUE TITLE
evecs.txt and projections.txt

### ISSUE THREAD

**Issue by liuhankui** [sriramlab/fastPPCA#10]
Hi, I have a problem with the result. I have 1267 SNPs from 100 samples. The geno-file is:
1267 100
1111000010101...
0101000101001...
0100100010000...
......

After calculation, I found that the evecs.txt contains 1267 rows and the projections.txt contains 100 rows. 
which file I should use for displaying the genetic structure of the 100 samples. I think I should use projections.txt, is that right?

**Comment by alecmchiu** [comment]
Yes, you should be using 'projections.txt' as these are the sample genotypes projected onto the principal components (evecs.txt).

**Comment by liuhankui** [comment]
> Yes, you should be using 'projections.txt' as these are the sample genotypes projected onto the principal components (evecs.txt).

Thanks for your help. I will use projections.txt. ProPCA is an excellent tool for big genotype matrix. Other tools, such as GCTA, generate principal components of sample genotypes into txt file with 'eigenvec' suffix. So, maybe there is a litter confusion when someone uses ProPCA instead of GCTA or plink. specifically when the ProPCA package storage principal components of sample genotypes in the example_evecs.txt in example-folder.
